gallery/src/components/NxBinaryDonutChart/NxBinaryDonutChartPage.tsx

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+
+import { GalleryDescriptionTile, GalleryExampleTile } from '../../gallery-components/GalleryTiles';
+import NxBinaryDonutChartMinimalExample from './NxBinaryDonutChartMinimalExample';
+import NxBinaryDonutChartNoHoleExample from './NxBinaryDonutChartNoHoleExample';
+import NxBinaryDonutChartLargeHoleExample from './NxBinaryDonutChartLargeHoleExample';
+
+const nxBinaryDonutChartMinimalExampleCode = require('!!raw-loader!./NxBinaryDonutChartMinimalExample').default;
+const nxBinaryDonutChartNoHoleExample = require('!!raw-loader!./NxBinaryDonutChartNoHoleExample').default;
+const nxBinaryDonutChartLargeHoleExample = require('!!raw-loader!./NxBinaryDonutChartLargeHoleExample').default;
+
+const NxBinaryDonutChartPage = () =>
+  <>
+    <GalleryDescriptionTile>
+      <p className="nx-p">
+        <code className="nx-code">NxBinaryDonutChart</code> represents a binary donut chart.
+      </p>
```

Comment:
Should have a table of props documentation like we do for other components.

Reply:
Added 2471b90185acb2beb99a29a9678dd6f7a1e99006